README.md

```diff
@@ -1,16 +1,47 @@
 skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=master)](https://travis-ci.org/projectatomic/skopeo)
 =
 
-_Please be aware that `skopeo` is still a work in progress and it currently supports only registry API V2_
+`skopeo` is a command line utility that performs various operations on container images and image repositories.  Skopeo works with API V2 registries such as Docker Registries, the Atomic Registry, Private Registries, local directories and local OCI-layout directories.  Skopeo does not require a daemon to be running to perform these operations which consist of:
```

Comment:
Should Docker _R_egistries, and _P_rivate _R_egistries be capitalized?

Reply:
Docker should always be capitalized unless you are talking about the CLI.

Reply:
Registries should not be,

Reply:
Good point.  I've kept 'Docker' and 'Atomic capitalized, and lower cased Registries and Registry on this line.